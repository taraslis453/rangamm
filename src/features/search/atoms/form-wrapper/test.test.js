import React from 'react'
import {render} from '@testing-library/react'
import {FormWrapper} from './index'
describe('<FormWrapper/>', () => {
  it('renders properly', () => {
    const {container} = render(<FormWrapper />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
