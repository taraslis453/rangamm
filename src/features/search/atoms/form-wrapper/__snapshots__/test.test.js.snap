// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormWrapper/> renders properly 1`] = `
<form
  class="sc-bdfBwQ dOKyap"
/>
`;
